### ISSUE TITLE
Getting Error while invoking lambda from API Gateway

### ISSUE THREAD

**Issue by jitesh-prajapati** [localstack/localstack#600]
```
Getting Error while invoking lambda from API Gateway
----------------------------------------------------------------
ERROR: Error forwarding request: the JSON object must be str, bytes or bytearray, not 'Response' Traceback (most recent call last):
  File "/usr/lib/python3.6/site-packages/localstack/services/generic_proxy.py", line 181, in forward
    path=path, data=data, headers=forward_headers)
  File "/usr/lib/python3.6/site-packages/localstack/services/apigateway/apigateway_listener.py", line 201, in forward_request
    parsed_result = result if isinstance(result, dict) else json.loads(result)
  File "/usr/lib64/python3.6/json/__init__.py", line 348, in loads
    'not {!r}'.format(s.__class__.__name__))
TypeError: the JSON object must be str, bytes or bytearray, not 'Response'
```

**Comment by NellyGzzA** [comment]
Any update about this? I just got the same error when trying to run Kinesis.

**Comment by whummer** [comment]
@jitesh-prajapati @NellyGzzA This should be fixed in #1502 - can you please give it a try with the `latest` version of the Docker image.

Closing this issue - please re-open if the problem persists. Thanks
